numba/lowering.py

```diff
@@ -691,169 +691,203 @@ def lower_call(self, resty, expr):
             fnty = self.typeof(expr.func.name)
 
         if isinstance(fnty, types.ObjModeDispatcher):
```

Comment:
Am I right in thinking that these large-ish changes are a simple refactor to put the bodies of the `isinstance` branches into functions?

Reply:
Yes